tests/msc2946_test.go

```diff
@@ -204,40 +204,7 @@ func TestClientSpacesSummary(t *testing.T) {
 	// - Rooms are returned correctly along with the custom fields `room_type`.
 	// - Events are returned correctly.
 	t.Run("query whole graph", func(t *testing.T) {
-		res := alice.MustDo(t, "GET", []string{"_matrix", "client", "unstable", "org.matrix.msc2946", "rooms", root, "spaces"}, nil)
-		must.MatchResponse(t, res, match.HTTPResponse{
```

Comment:
Why are we removing these checks?

Reply:
It doesn't seem useful to keep them around as this endpoint is deprecated and will be removed.